DTS-STN/dts-rmp
# 207 contact names in engagement list

# Description

With Marco's help, we can now display the names of the contacts in and engagement within a contact detail view card.

## Test Instructions

Navigate to search contacts and select one.
See at the bottom inside the engagements the contact names are displayed. 
Upon refresh they are still there.

## Help Requested


## Checklist
